apps/src/code-studio/initApp/project.js

```diff
@@ -368,16 +368,14 @@ var projects = (module.exports = {
   },
 
   /**
-   * Allows admin user to reset abuse score to either 0 or -50 (for buffer abuse score)
-   * and then saves the project.
+   * Allows admin user to reset abuse score to 0 and then saves the project.
    */
   adminResetAbuseScore(score = 0) {
```

Comment:
should `score` be a parameter here? It will always be set to 0 regardless of the parameter, right?

Reply:
Updated!